lcov.info

```diff
@@ -0,0 +1,7482 @@
+TN:
```

Comment:
no need to push this file

Reply:
[removed](https://github.com/lindy-labs/sandclock-contracts/pull/155/commits/07010700d3e1116d063b1303f8288df9865573bc)